ember-slugify/babel.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    "@babel/plugin-proposal-class-properties"
+  ]
+}
```

Comment:
Could you explain why it is needed?

Reply:
Could be nice to see if they are really needed on every addon or if we can remove them